ls1intum/Orion
# `Feature` Highlight Todos in Java Exercises

### Motivation and Context
Orion should be able to highlight todos in the template code even if the student deleted them.

### Description
the assessment mode of Orion now provides the ability to view todos as comments inside of the assessment editor.

### Steps for Testing
1. Install Orion and start an exercise in **assessment mode**.
2. Open a student file in the assessment editor and verify that the todos are mapped correctly.

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots

Newly added Files by the student do not show any todos
![grafik](https://github.com/ls1intum/Orion/assets/25010075/ed663626-d0e2-44bc-965a-acbeb656203b)

Todos can stick to the File, Class and Method
![grafik](https://github.com/ls1intum/Orion/assets/25010075/13244a7b-777e-4006-9e30-ec48231a693e)
![grafik](https://github.com/ls1intum/Orion/assets/25010075/e1f50dc9-946a-4143-bd66-c6f13ee51f4c)
